core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -101,8 +102,15 @@ protected DataBuffer getDataBuffer(BlockReadRequestContext context,
       openBlock(context, response);
       BlockReader blockReader = context.getBlockReader();
       Preconditions.checkState(blockReader != null);
-      ByteBuffer buf = blockReader.read(offset, len);
-      return new DataByteBuffer(buf, len);
+      ByteBuf buf = PooledByteBufAllocator.DEFAULT.buffer(len, len);
```

Comment:
Is the default allocator a good choice in all cases? Should we provide a way to configure the allocator (or maybe netty already does)?